src/App.jsx

```diff
@@ -6,28 +6,7 @@ function App() {
   const [count, setCount] = useState(0)
 
   return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src="/vite.svg" className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test 
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
+    <p>Hey</p>
   )
 }
 
```

Comment:
- [ ] Great work implementing your project this way. It shows you do your research. 👏. However, the requirements for this project state that you should have a class-based component called the `App` component and it should render the `Calculator` component (import the `Calculator` component here). Kindly implement it this way. 👍

Reply:
This change still need to be done